websites/L/Library Of Babel/metadata.json

```diff
@@ -0,0 +1,29 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.10",
+	"author": {
+		"id": "358402930191106049",
+		"name": "ion606"
+	},
+	"service": "Library Of Babel",
```

Comment:
You may need to use certain commands to rename the folder capitalization
```ts
	"service": "Library of Babel",
```

Reply:
wait what?

Reply:
The service is "Library **o**f Babel", rather than "Library **O**f Babel". If you could change the folder capitalization to fit the service, that would be preferable. 
https://stackoverflow.com/a/6899682/5844241 or https://stackoverflow.com/a/19200787/5844241 might be helpful

Reply:
oh my bad I didn't even notice that, thank you :3